Formula/nats-server.rb

```diff
@@ -12,10 +12,15 @@ class NatsServer < Formula
     sha256 "09b715a345599e63e3aa58c10523a27e9639d662753d7cca4b3fba7471432900" => :high_sierra
   end
 
-  depends_on "go" => :build
+  depends_on "go@1.13" => :build
```

Comment:
Why does the latest go version not work anymore?

Reply:
Here is the problem. We try to have the build here be the same than the official release that we have here: https://github.com/nats-io/nats-server/releases/tag/v2.1.7.
Ideally, I guess we would pull the server from there, but I am not sure how to write the formula to do so.
Something also that I wanted to discuss. I am part of the maintainer of NATS Server, and I notice that sometimes the formula is being updated by other people, which is nice, but it does not guarantee that people installing NATS Server from homebrew then have the "official" release. Is there a process for that? (for instance with Docker Hub, we submit a PR to official-images, and only the maintainers can do so).

Reply:
You're distributing it as open source and someone made a formula for it and everyone can update it if they see an improvement. The only way to stop that is change the license. 

Moving back to an outdated go version isn't an improvement though so that change will have to be reverted.

Reply:
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap#official-vendor-taps

Here's some more documentation on homebrews position.

Reply:
@SMillerDev We were simply trying to make consistent how we release the server and what version of Go we use to compile. We will upgrade but for this release we wanted to make it the same as our normal release for all of our release targets.